src/tasks/docker/package/run.js

```diff
@@ -13,10 +13,13 @@ const { buildContextEnvs } = require('KegUtils/builders/buildContextEnvs')
 const { mergeTaskOptions } = require('KegUtils/task/options/mergeTaskOptions')
 const { checkContainerExists } = require('KegUtils/docker/checkContainerExists')
 const { getImgInspectContext } = require('KegUtils/getters/getImgInspectContext')
+const { getOptsWithLabels } = require('KegUtils/docker/getOptsWithLabels')
 
 const { CONTAINER_PREFIXES, KEG_DOCKER_EXEC, KEG_EXEC_OPTS } = require('KegConst/constants')
 const { PACKAGE } = CONTAINER_PREFIXES
 
+const proxyHostKey = 'KEG_PROXY_HOST'
```

Comment:
Before it was being used in multiple places, but now it's only one place, so I'll just remove it actually.

Reply:
Since this is only being used once, and it's really a variable name, I actually just removed it. I think it would be one two many layers of indirection to have a constant mapping to a variable name mapping to the host url, especially when the variable name is internal and managed/determined by us.